src/styles/main.scss

```diff
@@ -0,0 +1,14 @@
+@mixin transition($properties...) {
+  transition: $properties $transition-duration;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: Roboto, sans-serif;
+  background-color: $white-color;
+}
```

Comment:
Ensure that the `$white-color` variable is defined in your variables file or elsewhere in your project. Without it, the background color will not be applied correctly.